app/themes/prebuilt/YoroiModern.js

```diff
@@ -136,6 +136,16 @@ const rpTextArea = {
   '--rp-textarea-text-color': '#353535',
 };
 
+// OPTION CARD
+const rpItemCard = {
+  '--rp-card-bg-color': '#fff',
```

Comment:
rp is a prefix for "ReactPolymorph" meant to indicte that this color is meant to be used as an override for the prebuilt components that come with React-Polymorph.
Since this is not the case for you, can you remove the rp prefix (both here and in the classic file)

Reply:
Fixed. Thanks